redash/query_runner/oracle.py

```diff
@@ -35,7 +35,7 @@ class Oracle(BaseSQLQueryRunner):
     @classmethod
     def get_col_type(cls, col_type, scale):
         if col_type == cx_Oracle.NUMBER:
-            return TYPE_FLOAT if scale > 0 else TYPE_INTEGER
+            return TYPE_INTEGER if scale is False else TYPE_FLOAT
```

Comment:
If `scale` can be `None`, then in such cases it will be classified as a float. Is this correct? If not, you might want to change the if statement.

Reply:
My understanding is that if scale is *not* `None` or is `> 0`  (so basically it is a truthy value) it should be classified as a Float, otherwise Integer.

However, here is the [definition ](https://cx-oracle.readthedocs.io/en/latest/user_guide/sql_execution.html#id10) from `cx_Oracle` for `NUMBER`:
```
If the column is unconstrained (no precision and scale specified), the value will be returned as a float or an int depending on whether the value itself is an integer. In all other cases the value is returned as a float.
```

Maybe we should look at the value it self whether it is an int or float to decide at this point?

Reply:
> Maybe we should look at the value itself whether it is an int or float to decide at this point?

Sounds like the safest option, no?